### DISCUSSION TITLE
Avatar image file

### DISCUSSION THREAD

**Discussion by leomanfredini** [vaga/hugo-theme-m10c#74] [Q&A] [answered]
Hi
Which path should I save the image file when using the `avatar` variable in `params` section?
I'm using the repository as a sub-module of github and the content of the theme directory is overwritten when deploy is done, deleting the files I copy.

**Comment by vaga** [comment, marked as answer]
You should add your avatar file in your `/static/` directory. (not in `/themes/m10c/static/`)
`avatar` parameter is `avatar.jpg` by default.

https://github.com/vaga/hugo-theme-m10c/blob/3094e3f118f9004d00880a03cd85e1c8bb295367/layouts/_default/baseof.html#L21

**Reply by dushyant03** [reply]
Hi, I have added a jpeg in the /static/images/some.jpeg, when I run this locally I can see the avatar, but when i push to github pages i get a 404. anything I am doing wrong here?
